### PR TITLE
updated MME conversion factors to AMDG guidelines. Needs to be converted to ELM

### PR DESCRIPTION
**Pull request by mldorr**
Only change is commenting out medications which are not included in the AMDG guidance. 

@achen2401 Could you do the quick conversion to ELM and then this can be pulled.

### CODE REVIEW

File: src/cql/r4/OMTKLogic.cql [hunk 1]
```diff
@@ -75,6 +79,29 @@ define function ToDaily(frequency Integer, period Quantity):
...
 /*
   Returns the conversion factor for the given ingredient
+Conversion factors are extrapolated from the 2015 AMDG Opioid Guideline, with conversion
+factors for the various forms of Fentanyl from the CDC and CMS guidelines listed below. 
+Note: some medications are not included in the AMDG guidelines and are thus not included
+in the conversion factors.
+
+Morphine Equivalent Dose for Selected Opioids from the 2015 AMDG Opioid Guideline
+http://www.agencymeddirectors.wa.gov/Files/2015AMDGOpioidGuideline.pdf
+Codeine 0.15
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:89]: codeine correct

File: src/cql/r4/OMTKLogic.cql [hunk 2]
```diff
@@ -75,6 +79,29 @@ define function ToDaily(frequency Integer, period Quantity):
...
+Fentanyl transdermal
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:90]: You have already sorted in new code, I suggest removing

File: src/cql/r4/OMTKLogic.cql [hunk 3]
```diff
@@ -75,6 +79,29 @@ define function ToDaily(frequency Integer, period Quantity):
...
+http://www.agencymeddirectors.wa.gov/Files/2015AMDGOpioidGuideline.pdf
+Codeine 0.15
+Fentanyl transdermal 
+Hydrocodone 1
+Hydromorphone 4
+Morphine 1
+Oxycodone 1.5
+Trapentadol 0.4
+Tramadol 0.1
+
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:97]: keep hydrocodone - tramadol, all correct

File: src/cql/r4/OMTKLogic.cql [hunk 4]
```diff
@@ -3,20 +3,24 @@ library OMTKLogic version '3.0.0'
 /*
 This library provides functionality for calculating Milligram Morphine
 Equivalents (MME) for opioid medications, as described in the CDC Opioid
-Prescribing Guideline.
+Prescribing Guideline *with the modifications made to meet the AMDG converstion standards*.
+
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:7]: keep

File: src/cql/r4/OMTKLogic.cql [hunk 5]
```diff
@@ -75,6 +79,29 @@ define function ToDaily(frequency Integer, period Quantity):
...
+Oxycodone 1.5
+Trapentadol 0.4
+Tramadol 0.1
+
+Methadone
+  1-20 mg/d 4
+  21-40 mg/d 8
+  41-60 mg/d 10
+  >60 mg/d 12
+
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:103]: These are the correct methadone conversion factors - keep

File: src/cql/r4/OMTKLogic.cql [hunk 6]
```diff
@@ -75,6 +79,29 @@ define function ToDaily(frequency Integer, period Quantity):
...
+
```
Comment (lydiadr) [inline src/cql/r4/OMTKLogic.cql:104]: as per slack message keep below and I will review against CDC table

**Comment by achen2401** [comment]
close out this PR in favor of [this updated one](https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/pull/81)

**Pull request closed without merging**